sirepo/template/srw.py

```diff
@@ -145,6 +145,8 @@
 
 _JSON_MESSAGE_EXPANSION = 20
 
+_MAX_MESSAGE_BYTES = sirepo.job.cfg().max_message_bytes
```

Comment:
Do not cache this.

Reply:
[851a07b](https://github.com/radiasoft/sirepo/pull/6186/commits/851a07bb4fdf6b2f875836aa9f0f36c01b0bd1cb)